exec/csnex/Exec.cs

```diff
@@ -50,8 +50,10 @@ public Executor(Module mod)
             init_order = new List<string>();
             module = mod;
             modules.Add("", mod);
+            Disassembler = new InstructionDisassembler(mod.Bytecode, ip);
             param_recursion_limit = 1000;
             Allocations = 0;
+            EnableTrace = csnex.gOptions.EnableTracing;
```

Comment:
What is the purpose of making a class member copy of this flag? Why not refer to `csnex.gOptions.EnableTracing` directly when needed?

Reply:
I wanted to make `Executor` self contained.  I fixed this by passing `csnex.gOptions.EnableTracing` as a constructor parameter.